guides/plugins/plugins/administration/add-new-tab.md

```diff
@@ -47,40 +64,66 @@ Having a look at the template, you might find the block `sw_product_detail_conte
+
+### Create custom tab
 
-Unfortunately, you cannot use the block mentioned above, because then your new tab wouldn't be inside the `<sw-tabs>` element. Instead, you can choose the last available block inside the element, which is `sw_product_detail_content_tabs_reviews`.
+Knowing the block you have to override in your plugin, you can now start doing exactly this: Add your custom tab by overriding this block called `sw_product_detail_content_tabs_reviews`.
 
-Knowing the block you have to override in your plugin, you can now start doing exactly this. First re-create the directory structure from the core code in your plugin. In this case, you'll have to create a directory structure like the following: `<plugin root>/src/Resources/app/administration/src/page/sw-product-detail`
+{% hint style="danger" %} However, please keep in mind that "overriding" doesn't mean we want to replace the block completely with our new one. We want to add our tab, thus only extending the template. This will have some implications on our implementation.
+{% endhint %}
+
+First, please re-create the directory structure from the core code in your plugin. In this case, you'll have to create a directory structure like the following: `<plugin root>/src/Resources/app/administration/src/page/sw-product-detail`
 
 In there you create a new file `index.js`, which then contains the following code:
 
+{% code title="<plugin root>/src/Resources/app/administration/src/page/sw-product-detail/index.js" %}
 ```javascript
 import template from './sw-product-detail.html.twig';
 
+// Override your template here, using the actual template from the core
 Shopware.Component.override('sw-product-detail', {
     template
 });
 ```
+{% endcode %}
 
-All it's doing, is to basically override the `sw-product-detail` component with a new template. The new template does not exist yet though, so create a new file `sw-product-detail.html.twig` in the same directory as your `index.js` file.
-
+All this file is doing is to basically override the `sw-product-detail` component with a new template. The new template does not exist yet though, so create a new file `sw-product-detail.html.twig` in the same directory as your `index.js` file.
 It then has to use the block we figured out earlier and override it by adding a new tab element:
 
-```markup
+{% code title="<plugin root>/src/Resources/app/administration/src/page/sw-product-detail/sw-product-detail.html.twig" %}
+```twig
+{% block sw_product_detail_content_tabs_reviews %}
+
+    {# This parent is very important as you don't want to override the review tab completely #}
+    {% parent %}
+
+{% endblock %}
+```
+{% endcode %}
+
+{% hint style="warning" %} The block gets overridden and immediately the parent block is called, since you do not want to replace the 'Review' tab, you want to add a new tab instead.
+{% endhint %}
```

Comment:
```suggestion
{% hint style="warning" %}
The block gets overridden and immediately the parent block is called, since you do not want to replace the 'Review' tab, you want to add a new tab instead.
{% endhint %}
```